apps/emqx_bridge_kafka/src/emqx_bridge_kafka.erl

```diff
@@ -678,9 +678,17 @@ kafka_connector_config_fields() ->
                 desc => ?DESC(metadata_request_timeout)
             })},
         {authentication,
-            mk(hoconsc:union([none, ref(auth_username_password), ref(auth_gssapi_kerberos)]), #{
-                default => none, desc => ?DESC("authentication")
-            })},
+            mk(
+                hoconsc:union([
+                    none,
+                    msk_iam,
```

Comment:
add this to desc.

Reply:
the desc current doesn't describe any of the union members :thinking:

Reply:
hm. i remembered it wrong.

great that you added it.